saleor/graphql/product/types/attributes.py

```diff
@@ -113,6 +113,10 @@ class Attribute(CountableDjangoObjectType):
         graphene.Boolean(description=AttributeDescriptions.FILTERABLE_IN_DASHBOARD),
         model_field="filterable_in_dashboard",
     )
+    available_in_grid = gql_optimizer.field(
```

Comment:
Why is this using the optimizer? I guess it's not required for a simple boolean field.